freeipa/src/main/java/com/sequenceiq/freeipa/flow/freeipa/verticalscale/FreeIpaVerticalScaleFlowConfig.java

```diff
@@ -0,0 +1,81 @@
+package com.sequenceiq.freeipa.flow.freeipa.verticalscale;
+
+import static com.sequenceiq.freeipa.flow.freeipa.verticalscale.FreeIpaVerticalScaleState.FINAL_STATE;
+import static com.sequenceiq.freeipa.flow.freeipa.verticalscale.FreeIpaVerticalScaleState.INIT_STATE;
+import static com.sequenceiq.freeipa.flow.freeipa.verticalscale.FreeIpaVerticalScaleState.STACK_VERTICALSCALE_FAILED_STATE;
+import static com.sequenceiq.freeipa.flow.freeipa.verticalscale.FreeIpaVerticalScaleState.STACK_VERTICALSCALE_FINISHED_STATE;
+import static com.sequenceiq.freeipa.flow.freeipa.verticalscale.FreeIpaVerticalScaleState.STACK_VERTICALSCALE_STATE;
+import static com.sequenceiq.freeipa.flow.freeipa.verticalscale.event.FreeIpaVerticalScaleEvent.STACK_VERTICALSCALE_EVENT;
+import static com.sequenceiq.freeipa.flow.freeipa.verticalscale.event.FreeIpaVerticalScaleEvent.STACK_VERTICALSCALE_FAILURE_EVENT;
+import static com.sequenceiq.freeipa.flow.freeipa.verticalscale.event.FreeIpaVerticalScaleEvent.STACK_VERTICALSCALE_FAIL_HANDLED_EVENT;
+import static com.sequenceiq.freeipa.flow.freeipa.verticalscale.event.FreeIpaVerticalScaleEvent.STACK_VERTICALSCALE_FINALIZED_EVENT;
+import static com.sequenceiq.freeipa.flow.freeipa.verticalscale.event.FreeIpaVerticalScaleEvent.STACK_VERTICALSCALE_FINISHED_EVENT;
+import static com.sequenceiq.freeipa.flow.freeipa.verticalscale.event.FreeIpaVerticalScaleEvent.STACK_VERTICALSCALE_FINISHED_FAILURE_EVENT;
+
+import java.util.List;
+
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.flow.core.config.AbstractFlowConfiguration;
+import com.sequenceiq.flow.core.config.AbstractFlowConfiguration.Transition.Builder;
+import com.sequenceiq.freeipa.flow.freeipa.verticalscale.event.FreeIpaVerticalScaleEvent;
+
+@Component
+public class FreeIpaVerticalScaleFlowConfig extends AbstractFlowConfiguration<FreeIpaVerticalScaleState, FreeIpaVerticalScaleEvent> {
+    private static final List<Transition<FreeIpaVerticalScaleState, FreeIpaVerticalScaleEvent>> TRANSITIONS =
+            new Builder<FreeIpaVerticalScaleState, FreeIpaVerticalScaleEvent>()
+                    .from(INIT_STATE)
+                    .to(STACK_VERTICALSCALE_STATE)
+                    .event(STACK_VERTICALSCALE_EVENT)
+                    .noFailureEvent()
+
+                    .from(STACK_VERTICALSCALE_STATE)
+                    .to(STACK_VERTICALSCALE_FINISHED_STATE)
+                    .event(STACK_VERTICALSCALE_FINISHED_EVENT)
+                    .failureEvent(STACK_VERTICALSCALE_FINISHED_FAILURE_EVENT)
```

Comment:
I still don't get why is the `STACK_VERTICALSCALE_FINISHED_FAILURE_EVENT` is here, and why not the `STACK_VERTICALSCALE_FAILURE_EVENT`